src/components/Button/ButtonWithAccessibleDisabledState.js

```diff
@@ -0,0 +1,19 @@
+import {Button as UtrechtButton} from '@utrecht/component-library-react';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// Temporary until the aria-disabled is set on the Utrecht button
+const ButtonWithAccessibleDisabledState = ({disabled, children, ...extraProps}) => {
+  return (
+    <UtrechtButton disabled={disabled} aria-disabled={disabled} {...extraProps}>
+      {children}
+    </UtrechtButton>
```

Comment:
This will also have the disabled attribute set :/

Reply:
I'm wondering if we instead should still have our own `Button` component that wraps `UtrechtButton`, but all it does is grab the `disabled` prop and turn it into `aria-disabled="true"`, install an `onClick` handler to prevent actual submitting and maybe manually set the disabled utrecht class name (if that's relevant).

So then we'd still everywhere import our "own" button, but it's just a thin shell addressing limitations in the upstream component.